README.md

```diff
@@ -46,21 +46,69 @@ Features include:
 Installing
 ==========
 
-There will _soon_ be a release of Swarm on Hackage and in a binary form!
-If you are interested in Haskell or functional programming, you can install
-it from source (see below) or, once it is released, with the Cabal tool.
-For instructions on how to install the binary directly, check the releases.
-
-Installing from Source
+**NOTE**: Swarm requires a POSIX-style terminal environment that
+supports `terminfo`.  Linux and MacOS should work out of the box.  On
+Windows, you will need to use [Windows Subsystem for
+Linux](https://learn.microsoft.com/en-us/windows/wsl/); you should
+then be able to follow instructions for installing on Linux.
+
+It is recommended that you use a relatively large terminal window
+(*e.g.* 170 columns x 40 rows or larger).  To find out the size of
+your terminal, you can type `stty size` at a command prompt. If it's
+not big enough, try decreasing the font size. You can read about
+and/or share recommended terminal settings in [this GitHub
+issue](https://github.com/swarm-game/swarm/issues/447).
+
+- [Installing via binaries](#installing-via-binaries)
+- [Installing from Hackage](#installing-from-Hackage)
+- [Installing from source](#installing-from-source)
+
+Installing via binaries
+-----------------------
+
+Currently we have one binary release built on an Ubuntu XXX system; it
+will probably work on any XXX.  We hope to add MacOS binaries in the
```

Comment:
@xsebek how would you fill in the `XXX`?

Reply:
Bionic and GNU/Linux.